src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProject.cs

```diff
@@ -179,6 +180,24 @@ private IList<string> GetConfigFilePaths(ISettings settings)
             return SettingsUtility.GetConfigFilePaths(settings).ToList();
         }
 
+        private void IgnoreUnsupportProjectReference(PackageSpec project)
+        {
+            foreach (var frameworkInfo in project.RestoreMetadata.TargetFrameworks)
+            {
+                var projectReferences = new List<ProjectRestoreReference>();
+
+                foreach (var projectReference in frameworkInfo.ProjectReferences)
+                {
+                    if (_projectSystemCache.ContainsKey(projectReference.ProjectUniqueName))
```

Comment:
What about non-SDK projects? Are they also in this cache?

Reply:
This also might end up doing way more than you expect, for example if the project system is slow to nominate a project you would end up fixing the problem here which isn't bad, but restore would end up being a subset of what it should be. It could end up running several times instead of the current behavior of failing and then nooping.

Reply:
This doesn't differentiate between a project being incompatible and a project not being nominated yet.

Reply:
Agree with Justin and Nikolche's point. This is not the right place to filter unsupported projects for net core.